apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Trace/CloudTraceExtension.cs

```diff
@@ -111,12 +111,19 @@ public static void UseGoogleTrace(this IApplicationBuilder app)
 
             services.AddSingleton<IManagedTracerFactory>(tracerFactory);
             services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
+            services.AddSingleton(CreateTraceHeaderPropagatingHandler);
```

Comment:
Any reason not to inline this as a lambda expression? Even though it's internal, I can't see you using it elsewhere.